lib/GLPI/Agent/Task/Inventory/Linux/AntiVirus/Bitdefender.pm

```diff
@@ -0,0 +1,87 @@
+package GLPI::Agent::Task::Inventory::Linux::AntiVirus::Bitdefender;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use UNIVERSAL::require;
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    return canRun('/opt/bitdefender-security-tools/bin/bduitool about');
```

Comment:
Don't put an option here. The agent won't run the command here but will check if the given command is runable (see [canRun() API](https://github.com/glpi-project/glpi-agent/blob/3acd88fb7d7df84d5231f09b9948c75d882fd059/lib/GLPI/Agent/Tools.pm#L559))... and in your case, this would be the full path to a command which contains a space and then doesn't exist. And so the module will never be enabled.

Reply:
OK. Done. 
Merci